packages/@webex/plugin-cc/src/services/index.ts

```
@@ -0,0 +1,43 @@
+    this.dialer = aqmDialer(aqmReq);
+    this.connectionService = new ConnectionService({
+      webSocketManager: this.webSocketManager,
+      subscribeRequest: connectionConfig,
+    });
+  }
+
+  public static getInstance(options: {
+    webex: WebexSDK;
+    connectionConfig: SubscribeRequest;
+  }): Services {
+    if (!this.instance) {
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)